tests/account.test.js

```diff
@@ -0,0 +1,46 @@
+// account.test.js
+import { describe, it, expect } from 'vitest';
+import { Account } from '../src/model/Account.js';
+
+describe('Account', () => {
+  it('Criacao de uma conta valida', () => {

```

Comment:
sadly, our language is not commonly used for test documentation, so I strongly suggest using english 🇬🇧 for test descriptions